lib/base-service.ts

```diff
@@ -158,22 +170,22 @@ export class BaseService {
   /**
    * Wrapper around `sendRequest` that enforces the request will be authenticated.
    *
-   * @param {object} parameters - service request options passed in by user
-   * @param {string} parameters.options.method - the http method
-   * @param {string} parameters.options.url - the path portion of the URL to be appended to the serviceUr
-   * @param {object} [parameters.options.path] - the path parameters to be inserted into the URL
-   * @param {object} [parameters.options.qs] - the querystring to be included in the URL
-   * @param {object} [parameters.options.body] - the data to be sent as the request body
-   * @param {object} [parameters.options.form] - an object containing the key/value pairs for a www-form-urlencoded request
-   * @param {object} [parameters.options.formData] - an object containing the contents for a multipart/form-data request
+   * @param {object} parameters Service request options passed in by user.
+   * @param {string} parameters.options.method The http method.
+   * @param {string} parameters.options.url The path portion of the URL to be appended to the serviceUrl.
+   * @param {object} parameters.options.path The path parameters to be inserted into the URL.
+   * @param {object} parameters.options.qs The querystring to be included in the URL.
+   * @param {object} parameters.options.body The data to be sent as the request body.
+   * @param {object} parameters.options.form An object containing the key/value pairs for a www-form-urlencoded request.
+   * @param {object} parameters.options.formData An object containing the contents for a multipart/form-data request
```

Comment:
All the params that had brackets should maintain them - they are technically optional

Reply:
Ok I'll add it back.